tests/Unit/MailjetMailerTestCase.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Unit;
+
+use Http\Discovery\Psr17Factory;
+use Http\Mock\Client;
+use PHPUnit\Framework\TestCase;
+use SubscribeMe\Subscriber\MailjetSubscriber;
+use SubscribeMe\ValueObject\EmailAddress;
+
+class MailjetMailerTestCase extends TestCase
+{
+    public function testSubscribe(): void
+    {
+        $client = new Client();
+        $factory = new Psr17Factory();
+        $mailjetSubscriber = new MailjetSubscriber($client, $factory, $factory);
+
+        $options = [
+            'Name' => 'Passenger 1',
+        ];
+
+        $mailjetSubscriber->subscribe("passenger@mailjet.com", $options);
```

Comment:
Where are the credentials ?

This method should throw an exception because no credentials ?